lib/bwu_datagrid.dart

```diff
@@ -201,6 +203,9 @@ class BwuDatagrid extends PolymerElement {
 
   void _init() {
     _container = this.shadowRoot;
+    _instanceID = this.id != null && this.id != ""
```

Comment:
`this` to refer instance variables is discouraged in Dart except if it is necessary for disambiguation. I didn't find anything about it in the style guide though and the existing code base isn't consistent because the original JS uses `this` everywhere because it's necessary in JS. I usually remove it when I touch code.

`id.isNotEmpty` is preferred to `id != ''`